docs/contributing.rst

```diff
@@ -1,10 +1,535 @@
-Contributing to SPRAS
-=====================
+SPRAS contribution guide
+========================
 
-We welcome contributions to SPRAS! If you would like to contribute, please follow the guidelines below.
+The SPRAS maintainers welcome external contributions. Code contributions
+will be licensed using the project’s `MIT license <LICENSE>`__.
```

Comment:
Thanks to the new docs preview, I could check that relative links like this will break.  I didn't find any others.